### ISSUE TITLE
Bee Swarm Simulador infinite loading screen.

### ISSUE THREAD

**Issue by PedriNH4x** [vinegarhq/sober#191]
### Acknowledgement of checking existent issues

- [X] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

![20240903_100348](https://github.com/user-attachments/assets/223512e4-33aa-478f-af6c-b929208ae569)

the game Just freeze on the loading screen.

### Roblox Experience link

roblox.com/games/1537690962/Bee-Swarm-Simulator

**Comment by PedriNH4x** [comment]
https://github.com/user-attachments/assets/59a0e141-d393-4bf1-a005-2af9e6cecaa5

the error is not resolved after 4 sober updates.

**Comment by TymekBrunka** [comment]
not just bee swarm, and you can "play it" but you still see loading screen

**Comment by PedriNH4x** [comment]
> not just bee swarm, and you can "play it" but you still see loading screen

That's right, but the loading screen is in your face and you can't see anything that happens.
